cmake/ctest/drivers/atdm/ats2/drivers/Trilinos-atdm-ats2-cuda-10.1.243-gnu-7.3.1-spmpi-2019.06.24_static_opt.sh

```diff
@@ -2,4 +2,11 @@
 if [ "${Trilinos_TRACK}" == "" ] ; then
   export Trilinos_TRACK=Experimental
 fi
+export TPETRA_ASSUME_CUDA_AWARE_MPI=0
 $WORKSPACE/Trilinos/cmake/ctest/drivers/atdm/ats2/local-driver.sh
```

Comment:
Should we not also be configuring, building, and running on the compute node as well, even for the non-CUDA aware MPI mode?  That way we can use the entire node to build and run.

Reply:
Right, I've done this on a per-project basis from jenkins -- checkout the "Execute Shell" section for the nightly ATS2 jobs.